src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/CommandsPane.cs

```diff
@@ -113,11 +111,11 @@ public override int GetOptimalHeight(int width)
         /// <inheritdoc />
         internal override bool SupportsUiaProviders => true;
 
-        private void LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
+        private void LinkClicked(object? sender, LinkLabelLinkClickedEventArgs e)
         {
             try
             {
-                if (!e.Link.Enabled)
+                if (!e.Link!.Enabled)
```

Comment:
This may result in a NRE, we can't do this.
```suggestion
                if (e.Link is null || !e.Link.Enabled)
```